codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/DafnyLocalServiceTypeConversionProtocol.java

```diff
@@ -161,28 +158,41 @@ public void generateSerializers(final GenerationContext context) {
                 ),
               SmithyNameResolver.shapeNamespace(serviceShape),
               writer -> {
-                writer.addImportFromModule(
-                  SmithyNameResolver.getGoModuleNameForSmithyNamespace(
-                    output.toShapeId().getNamespace()
-                  ),
-                  SmithyNameResolver.smithyTypesNamespace(output)
-                );
-                writer.write(
-                  """
-                  func $L(nativeOutput $L)($L) {
-                      ${C|}
-                  }""",
-                  outputToDafnyMethodName,
-                  SmithyNameResolver.getSmithyType(output, outputSymbol),
-                  DafnyNameResolver.getDafnyType(output, outputSymbol),
-                  writer.consumer(w ->
-                    generateResponseSerializer(
+                if (output.hasTrait(PositionalTrait.class)) {
+                  output.accept(
```

Comment:
I need conversion function of memberShape of shape with positional trait to be written. So, I am visiting the shape visitor.

Reply:
And you skip generating the top-level `func` I guess?

Reply:
Yes. Don't write Top level shape but write memberShape conversion